doc/lua_api.txt

```diff
@@ -1837,6 +1837,7 @@ Spatial Vectors
 * `vector.round(v)`: returns a vector, each dimension rounded to nearest int
 * `vector.apply(v, func)`: returns a vector
 * `vector.equals(v1, v2)`: returns a boolean
+* `vector.sort(v1, v2)`: returns two vectors creating a cube that contains v1 and v2
```

Comment:
Returns in order the minimum and maximum points of the cuboid defined by v1, v2.
Or something.